doc/source/whatsnew/v0.17.0.txt

```diff
@@ -87,9 +87,12 @@ Backwards incompatible API changes
 
 .. _whatsnew_0170.api_breaking.other:
 
-.. _whatsnew_0170.api_breaking.convert_objects:
+
 Changes to convert_objects
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _whatsnew_0170.api_breaking.convert_objects:
```

Comment:
Is there a reason you changed this? 
Normally, the link label is typically put before the header (although we are not always consistent in the pandas docs) http://sphinx-doc.org/markup/inline.html#ref-role